### PR TITLE
Update external documentation URLs

### PR DESCRIPTION
**Pull request by bondjimbond**
**JIRA Ticket**: (https://jira.duraspace.org/browse/ISLANDORA-2034)

# What does this Pull Request do?
Updates references to the Apache Solr Wiki to its new location (which changed June 2017)

# What's new?
External links to the documentation on Solr fields, specifically Sort and DisMax, updated to the new location of the Solr guide.

# How should this be tested?

Go to the Solr config pages and make sure that the external links go to the right places.

# Interested parties
@Islandora/7-x-1-x-committers

### CODE REVIEW

**Comment by bondjimbond** [comment]
An additional improvement would be to put these URLs into a single variable that could be referenced by all of these different .inc files. The same link to the Sort documentation had to be changed in several different files; would be nice to be able to update it in just one place.

**Comment by bondjimbond** [comment]
Thanks, @rosiel! I'm still getting used to these things.

**Comment by DiegoPino** [comment]
I would suggest pointing just to this folder http://archive.apache.org/dist/lucene/solr/ref-guide/
and point them to supported /tested really versions 
like http://archive.apache.org/dist/lucene/solr/ref-guide/apache-solr-ref-guide-4.10.pdf
etc. But making it lucene.apache.org/solr/guide/6_6 can confuse a bit, since it "implies" 6.x is tested and well, not.

**Comment by DiegoPino** [comment]
@bondjimbond any updates on this? @rosiel do you have some thoughts? Are you all good with pointing to version 6.x?

**Comment by rosiel** [comment]
How about rewording it along the lines of:
`'For more information, see "sort" as discussed in the "Common query parameters" 
section of the <a href="!url">Solr documentation</a>.',
 array('!url' => 'http://archive.apache.org/dist/lucene/solr/ref-guide/'))`

I agree that we shouldn't point to 6 as a version if it's not tested yet. It's silly in a way because Common Query Parameters don't change much, but this will also future-proof it, because from the looks of it the 6_6 documentation will be bundled in a PDF as soon as the next version is released.

File: includes/admin.inc [hunk 1]
```diff
@@ -618,8 +618,8 @@ function islandora_solr_admin_settings($form, &$form_state) {
     '#type' => 'textarea',
     '#title' => t('Query fields'),
     '#default_value' => variable_get('islandora_solr_query_fields', 'dc.title^5 dc.subject^2 dc.description^2 dc.creator^2 dc.contributor^1 dc.type'),
-    '#description' => t('<a href="@url" target="_blank" title="Solr query fields documentation">Query fields</a> to use for DisMax (simple) searches.', array(
-      '@url' => 'http://wiki.apache.org/solr/DisMaxQParserPlugin#qf_.28Query_Fields.29',
+    '#description' => t('Query fields to use for DisMax (simple) searches. See the QueryFields section in the <a href="@url" target="_blank" title="Solr documentation">Solr documentation</a> for examples.', array(
```
Comment (DiegoPino) [inline includes/admin.inc:621]: Since you are here. "DisMax or eDisMax"?

Comment (DiegoPino) [reply includes/admin.inc:621]: Its a question really, dismax and edismax, both are valid options. And if you are using edismax, those apply too. But not sure if that should go there?

Comment (bondjimbond) [reply includes/admin.inc:621]: DisMax is what was originally there... knowing the difference is beyond my pay grade.

Comment (DiegoPino) [reply includes/admin.inc:621]: man, you get actually paid! 😆  Ok, but people managing a repository and using that could know the difference? https://cwiki.apache.org/confluence/display/solr/The+Extended+DisMax+Query+Parser . Not a blocker anyway

File: includes/blocks.inc [hunk 2]
```diff
@@ -885,8 +885,9 @@ function islandora_solr_sort() {
       $current_sort_order = isset($sort_explode[1]) ? $sort_explode[1] : NULL;
     }
     else {
-      // If no parameters are given, internally solr defaults to 'score desc'
-      // http://wiki.apache.org/solr/CommonQueryParameters#sort
+      // If no parameters are given, internally solr defaults to 'score desc'.
```
Comment (rosiel) [inline includes/blocks.inc]: Travis got angry because of whitespace here. Other than that, looks good!

**Review by rosiel — CHANGES_REQUESTED** [review]

**Review by rosiel — APPROVED** [review]

**Comment by DiegoPino** [comment]
@bondjimbond ping. Can you fix this before code freeze? https://travis-ci.org/Islandora/islandora_solr_search/jobs/272500140#L704 Thanks

**Comment by DiegoPino** [comment]
@bondjimbond ping. Please: Code freeze is tomorrow. Could you get the DCS issue fixed so we can merge? Your changes are a good contribution to this new release. Thanks!

**Comment by bondjimbond** [comment]
@DiegoPino I'm confused for two reasons... first I never modified that line, and the whitespace does not show up when I edit the file. Even if I try to edit directly in Github, there's no whitespace on that line to delete.

**Comment by rosiel** [comment]
Line 888's got your space at the end of the line. https://github.com/bondjimbond/islandora_solr_search/blob/issue-2034/includes/blocks.inc#L888 (from that link, you can use Github's edit feature if you want)

**Comment by whikloj** [comment]
@bondjimbond I opened a PR against your PR to clean this up in case that helps.

**Comment by bondjimbond** [comment]
@rosiel Thanks... why did Travis say it was line 932??

Thanks @whikloj for the PR to fix it while I wasn't looking.

**Comment by rosiel** [comment]
@bondjimbond I don't know where Travis got its line numbering from. It's not usually that off. Thanks for fixing!
